design/ceph/ceph-cluster-cleanup.md

```diff
@@ -7,8 +7,11 @@ As a rook user, I want to clean up data on the hosts after I intentionally unins
 ## Background
 
 ### Host-based cluster deletion
+If the user deletes a host-based cluster (running on raw devices and not on PVs) and starts a new cluster on the same hosts, then the user needs to perform the following manual steps:
+- Delete the  dataDirHostPath on each hosts. Otherwise, stale keys and other config will remain from the previous cluster and the new mons will fail to start.
```

Comment:
```suggestion
- Delete the dataDirHostPath on each host. Otherwise, stale keys and other configs will remain from the previous cluster and the new mons will fail to start.
```